app/invocation/invocation.tsx

```diff
@@ -251,6 +252,14 @@ export default class InvocationComponent extends React.Component<Props, State> {
             <ExecutionCardComponent model={this.state.model} inProgress={this.state.inProgress} />
           )}
 
+          {activeTab == "action" && (
+            <ActionCardComponent
+              model={this.state.model}
+              inProgress={this.state.inProgress}
```

Comment:
doesn't look like `inProgress` is used by the component? (the component has its own internal loading state, so this could be removed)